### ISSUE TITLE
OpenSea shows an authentication error after attempting to sign a message

### ISSUE THREAD

**Issue by 0xDaedalus** [tallycash/extension#1251]
It looks like the application is getting `undefined` back when calling `personal_sign`.

https://user-images.githubusercontent.com/94649004/160132624-b9df1cb9-d401-4d57-a2e6-0e4a25e6ce58.mov

```
 content: inpage > background: {"id":"102","target":"tally-provider-bridge","request":{"method":"personal_sign"
 ```
 
 ```
 instrument.js:109 Error: Unexpected JSON-RPC response jsonrpc. Expected: 2.0; Got: undefined
 ```

**Comment by greg-nagy** [comment]
@SJQuality can you please verify if this is still an issue for us? 

If it still persists then the problem might be that we are not returning the json rpc version in the response.

**Comment by SJQuality** [comment]
@greg-nagy So, connecting the wallet initially on OpenSea worked just fine. But then when I tried what's in the video (making an offer) it didn't work. So, I logged out of OpenSea and tried what's in the video and it failed also. I then tried to just connect to OpenSea like I did originally. That fails now too. So, I uninstalled Tally and reinstalled. Now either simply connecting to OpenSea or Making an offer fails. 

![image](https://user-images.githubusercontent.com/103908695/172398905-6b746234-22f0-4f5d-bdaf-07222bcfb3a1.png)
